discovery/operational_questions.md

```diff
@@ -0,0 +1,11 @@
+# Operational Questions
+## Summary
+This document will catalogue the questions and problems that can be answered/solved using data from the [minutes of the Board of Estimates](https://comptroller.baltimorecity.gov/boe/meetings/minutes).
+
+## Questions
+1. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+1. Phasellus ornare facilisis semper.
+
+## Problems
```

Comment:
I am not clear on what the difference between a question and a problem would be, when using this document. 

For example, a question could be: "How many budget account numbers does the BoE make decisions on each year?" 

How is  a problem different from that?